tests/plugins/__snapshots__/renderer.test.ts.snap

```diff
@@ -1296,7 +1296,7 @@ Array [
   "context.lineWidth=0.4;",
   "context.strokeRect(864.04, 966.04, 95.92, 22.92)",
   "context.textBaseline=\\"top\\";",
-  "context.font=\\"400 13px 'Roboto', arial\\";",
+  "context.font=\\"400 13.3px 'Roboto', arial\\";",
```

Comment:
Not sure what changed this value.
That means the default font size also changed?
Is it because of the computation in `fontSizeInPixels`?
Could we round the result ? I'm not sure what side-effect decimal pixels can have.

Reply:
Yes, this was due to the new computation. Adding a Math.round give back the previous values used in the tests/snapshots :)